api/db/seeds/data/certification/certification-courses-builder.js

```diff
@@ -33,6 +38,7 @@ const ASSESSMENT_FAILURE_PUBLISHED_SESSION_ID = 108;
 const ASSESSMENT_SUCCESS_PUBLISHED_SESSION_SCO_ID = 109;
 const CERTIFICATION_COURSE_SUCCESS_ID = 200;
 const CERTIFICATION_COURSE_FAILURE_ID = 403;
+const ASSESSMENT_SUCCESS_IN_PRO_SESSION_TO_FINALIZE_ID = 110;
```

Comment:
suggestion: remettre dans l'ordre
```suggestion
const ASSESSMENT_SUCCESS_PUBLISHED_SESSION_SCO_ID = 109;
const ASSESSMENT_SUCCESS_IN_PRO_SESSION_TO_FINALIZE_ID = 110;
const CERTIFICATION_COURSE_SUCCESS_ID = 200;
const CERTIFICATION_COURSE_FAILURE_ID = 403;
```